### ISSUE TITLE
Cupertino style

### ISSUE THREAD

**Issue by ricardoboss** [ueman/feedback#279]
Great package!

**Is your feature request related to a problem? Please describe.**
I am developing a Cupertino-style app and would like to use feedback. Unfortunately, feedback looks like Material and I cannot control enough of the style to make it look native.

**Describe the solution you'd like**
I'd like to have a way to either make it look like Cupertino or have enough configuration options to make it look like Cupertino myself.

**Describe alternatives you've considered**
One alternative may be to just let it look like Material and just change the colors via the theme option.

**Comment by ueman** [comment]
Hey, do you have any mockups or examples how this could look like when it's cupertino styled? I'm having troubles thinking of a design that looks Cupertino-y.

**Comment by ueman** [comment]
I‘m guessing the screenshot edit screen would be a good starting point 
![IMG_0058](https://github.com/ueman/feedback/assets/1270149/c8843d02-e738-440a-9e2c-cff0b32ae2b4)

**Comment by ricardoboss** [comment]
> Hey, do you have any mockups or examples how this could look like when it's cupertino styled?

No, unfortunately not. I like the idea of the screenshot edit screen though! Maybe I can provide guidance for how to apply Apples HID guidelines

**Comment by ricardoboss** [comment]
A colleague pointed me to this repo, which is apparently a Swift feedback package. It has this screenshot, which shows its design:

![](https://github.com/Lickability/PinpointKit/blob/master/Assets/screenshots.png?raw=true)

**Comment by ricardoboss** [comment]
[Bugsee](https://www.bugsee.com/) has this design:

![image](https://github.com/ueman/feedback/assets/6266356/fe9facb2-b6bf-4f29-95c0-94f69cc83802)

**Comment by ricardoboss** [comment]
Maybe, instead of overhauling the whole design just for cupertino, a good first step might be to add callbacks to allow customizing each part of the feedback screen.

I am thinking of `buildSubmitButton`, `buildDeleteButton` and `buildDrawButton` options to customize each of these widgets. Maybe also add a way to customize the icons, since I am not using any Material icons, except for `feedback`.

**Comment by chucker** [comment]
> I‘m guessing the screenshot edit screen would be a good starting point

Some of this UI is available in PencilKit. Looks like a Flutter wrapper exists: https://pub.dev/packages/pencil_kit
